linkerd/core/src/main/scala/io/buoyant/linkerd/NamerInitializer.scala

```diff
@@ -38,4 +38,6 @@ trait NamerConfig {
   def newNamer(): Namer
 }
 
-abstract class NamerInitializer extends ConfigInitializer
+abstract class NamerInitializer extends ConfigInitializer {
+  override def configId = configClass.getName
```

Comment:
Should we move this default impl up into ConfigInitializer and just override it in ProtocolInitializer?